tools/deployment/package-helm/templates/results-cache-statefulset.yaml

```
@@ -56,6 +56,32 @@ spec:
+            - |
+              until python3 -c "import socket; s=socket.socket(); s.settimeout(2); s.connect(('127.0.0.1', 27017)); s.close()" 2>/dev/null; do
+                sleep 1
+              done
+              python3 -u -m clp_py_utils.initialize-results-cache \
+                --uri "mongodb://127.0.0.1:27017/{{ .Values.clpConfig.results_cache.db_name }}" \
+                --member-host "{{ include "clp.fullname" . }}-results-cache:27017" \
+                --stream-collection {{ .Values.clpConfig.results_cache.stream_collection_name | quote }}
+              touch /tmp/init-complete
+              sleep infinity
+          readinessProbe:
+            {{- include "clp.readinessProbeTimings" . | nindent 12 }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)